lms/djangoapps/branding/api.py

```diff
@@ -25,6 +25,7 @@
 from branding.models import BrandingApiConfig
 from edxmako.shortcuts import marketing_link
 from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
+from requests.compat import urljoin
```

Comment:
Use six to import urljoin `from six.moves.urllib.parse import urljoin`

Reply:
Changed the `urljoin` module import statement.